internal/util/util.go

```diff
@@ -418,3 +420,18 @@ func KeysToSlice(m map[string]bool) []string {
 	sort.Strings(s)
 	return s
 }
+
+// ParseBigtableGroup retrieves Bigtable names for import group from
+// newline separated list.
+func ParseBigtableGroup(s string) []string {
```

Comment:
If using "comma separated list", this can be:

importGroups := strings.Split(s, ",")

I think that's another point to better use comma instead of newline to separate. Because newline can mean "\n", "\r", "\n\r", etc. But comma just means ",".

Reply:
That was my early concern, but seems like readLine might handle different newlines?  I do like the comma-seperated suggestion too.

Reply:
The reason I use newline separated format is for ease of manual editing. Imagine the version file contains 20 tables, it would be hard to update selected ones, and that's the only way for us to update the file.